lib/maze/hooks/appium_hooks.rb

```diff
@@ -27,6 +27,9 @@ def before_all
                                                    config.username,
                                                    config.access_key
         when :local
+          # Open the app beforehand to ensure it runs correctly and is accessible
+          Maze::Runner.run_command("/System/Library/Frameworks/CoreServices.framework/Versions/Current/Frameworks/LaunchServices.framework/Versions/Current/Support/lsregister -f -R -trusted #{config.app_path}") if config.os == 'macos'
```

Comment:
I've tested this locally and found that, unfortunately, this step alone is not enough - macOS continues to open the "older" version of the fixture 😞 

Adding another step to reset the LaunchServices database works around this, but it unregisters **all** apps and so could cause trouble elsewhere.

```
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/Support/lsregister -kill -r -v -apps u,s,l
```

Another approach we could take is to define a `Maze.driver.launch_app` method for macOS (I believe this is currently not implemented) that simply runs `open #{config.app_path}"`

Reply:
The best documentation I've found for `lsregister` is https://eclecticlight.co/2019/03/25/lsregister-a-valuable-undocumented-command-for-launchservices/

Reply:
> Another approach we could take is to define a `Maze.driver.launch_app` method for macOS (I believe this is currently not implemented) that simply runs `open #{config.app_path}"`

If that works I think it would clean up the code in the cocoa repo a bit?

Reply:
> Another approach we could take is to define a Maze.driver.launch_app method for macOS (I believe this is currently not implemented) that simply runs open #{config.app_path}"
I tried this, and didn't think it had worked.  I'll make the change, if you could test it afterwards that'd be great (I'll let you know on slack)

> > Another approach we could take is to define a `Maze.driver.launch_app` method for macOS (I believe this is currently not implemented) that simply runs `open #{config.app_path}"`
> 
> If that works I think it would clean up the code in the cocoa repo a bit?

How do you mean, removing the unzipping process in `env.rb`?

Reply:
I just meant a couple of places where we call `launch_app` on iOS and do something else on macOS, but it may not apply in those situation to be honest.  E.g. step `I relaunch the app after a crash`.

Reply:
This would only be relevant before launching the app for the first time, rather than every re-launch